build/env_default.go

```diff
@@ -34,6 +34,13 @@ var (
 		MigrationSurchargeMultiplier:  10,                                                  // 10x
 	}
 
+	// DefaultPricePinSettings define the default price pin settings the bus is
+	// configured with on startup. These values can be adjusted using the
+	// settings API.
+	DefaultPricePinSettings = api.PricePinSettings{
```

Comment:
We default all our settings. We can not set it this time because you don't really need a default here. I figured the trade-off would be that a) it's consistent with other settings and b) it's kind of nice if you call `/settings` you get all the bus setting keys. I'd usually remove the default here but especially `b` is kind of worthwhile? Don't mind removing it though and not setting it...